modules/nf-core/gtdbtk/classifywf/meta.yml

```diff
@@ -31,6 +31,10 @@ input:
       type: file
       description: The local copy of the taxonomic database used by GTDB-tk (unzipped copy)
       pattern: "*"
+  - mash_db:
+      type: file
+      description: The local copy of the Mash sketch database used by GTDB-tk (optional)
```

Comment:
For which contexts?

Reply:
As in, why is it optional - in which cases would you want to supply such a database

Reply:
https://ecogenomics.github.io/GTDBTk/commands/classify_wf.html#mutually%20exclusive%20required%20arguments_repeat1